rtgui/icmpanel.cc

```diff
@@ -184,51 +196,187 @@ ICMPanel::ICMPanel() : FoldableToolPanel(this, "icm", M("TP_ICM_LABEL")), iuncha
+    will->append(M("TP_ICM_WORKING_ILLU_D50"));
+    will->append(M("TP_ICM_WORKING_ILLU_D55"));
+    will->append(M("TP_ICM_WORKING_ILLU_D60"));
+    will->append(M("TP_ICM_WORKING_ILLU_D65"));
+    will->append(M("TP_ICM_WORKING_ILLU_D80"));
+    will->append(M("TP_ICM_WORKING_ILLU_D120"));
+    will->append(M("TP_ICM_WORKING_ILLU_STDA"));
+    will->append(M("TP_ICM_WORKING_ILLU_2000"));
+    will->append(M("TP_ICM_WORKING_ILLU_1500"));
+    will->set_active(0);
+    will->set_tooltip_text(M("TP_ICM_ILLUMPRIM_TOOLTIP"));
+
+
+    wprimBox = Gtk::manage(new Gtk::Box());
+    wprimlab = Gtk::manage(new Gtk::Label(M("TP_ICM_WORKING_PRIM") + ":"));
+
+    wprimBox->pack_start(*wprimlab, Gtk::PACK_SHRINK);
+    wprim = Gtk::manage(new MyComboBoxText());
+    wprimBox->pack_start(*wprim, Gtk::PACK_EXPAND_WIDGET);
+    trcProfVBox->pack_start(*wprimBox, Gtk::PACK_EXPAND_WIDGET);
+    wprim->append(M("TP_ICM_WORKING_PRIM_NONE"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_SRGB"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_ADOB"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_PROP"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_REC"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_ACE"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_WID"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_AC0"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_BRU"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_BET"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_BST"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_CUS"));
+    wprim->append(M("TP_ICM_WORKING_PRIM_CUSGR"));
+    wprim->set_active(0);
+
+    wprim->set_tooltip_text(M("TP_ICM_PRIMILLUM_TOOLTIP"));
+
+
+    redx = Gtk::manage(new Adjuster(M("TC_PRIM_REDX"), 0.41, 1.0, 0.0001, 0.6400));
+    setExpandAlignProperties(redx, true, false, Gtk::ALIGN_FILL, Gtk::ALIGN_CENTER);
+    redy = Gtk::manage(new Adjuster(M("TC_PRIM_REDY"), 0.0, 0.70, 0.0001, 0.3300));
+    setExpandAlignProperties(redy, true, false, Gtk::ALIGN_FILL, Gtk::ALIGN_CENTER);
+    grex = Gtk::manage(new Adjuster(M("TC_PRIM_GREX"), -0.1, 0.4, 0.0001, 0.2100));
+    setExpandAlignProperties(grex, true, false, Gtk::ALIGN_FILL, Gtk::ALIGN_CENTER);
+    grey = Gtk::manage(new Adjuster(M("TC_PRIM_GREY"), 0.50, 1.0, 0.0001, 0.7100));
+    setExpandAlignProperties(grey, true, false, Gtk::ALIGN_FILL, Gtk::ALIGN_CENTER);
+    blux = Gtk::manage(new Adjuster(M("TC_PRIM_BLUX"), -0.1, 0.4, 0.0001, 0.1500));
+    setExpandAlignProperties(blux, true, false, Gtk::ALIGN_FILL, Gtk::ALIGN_CENTER);
+    bluy = Gtk::manage(new Adjuster(M("TC_PRIM_BLUY"), -0.1, 0.5, 0.0001, 0.060));
+    setExpandAlignProperties(bluy, true, false, Gtk::ALIGN_FILL, Gtk::ALIGN_CENTER);
+
+    redx->set_tooltip_text(M("TP_ICM_PRIMRED_TOOLTIP"));
+    grex->set_tooltip_text(M("TP_ICM_PRIMGRE_TOOLTIP"));
+    blux->set_tooltip_text(M("TP_ICM_PRIMBLU_TOOLTIP"));
+    blr = Gtk::manage(new Gtk::Label(M(" ")));
+    blg = Gtk::manage(new Gtk::Label(M(" ")));
+    blb = Gtk::manage(new Gtk::Label(M("   ")));
```

Comment:
Is this double space intentional?
I don't know exactly what this is for, but handling spacing in the GUI through whitespace in a label seems a little strange. Maybe not so bad atm.

Reply:
yes because somme sliders have negatif values